### DISCUSSION TITLE
How to cite MyST-nb

### DISCUSSION THREAD

**Discussion by NatalieZelenka** [executablebooks/meta#644] [Q&A] [answered]
Hello I'd like to cite MyST-nb https://myst-nb.readthedocs.io/en/latest/ 

Is there a specific way that would be good?

**Comment by choldgraf** [comment, marked as answer]
Hmm - we don't have a citation specifically for MyST-NB, but there is a Zenodo record for Jupyter Book. Would that work? https://zenodo.org/record/4539666

**Comment by NatalieZelenka** [comment]
Sure I can cite that instead!
